zgrab2/multiple.go

```diff
@@ -0,0 +1,31 @@
+package zgrab2
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ajholland/zflags"
+)
+
+type MultipleConfig struct {
+	ConfigFileName  string `short:"c" long:"config-file" default:"-" description:"Config filename, use - for stdin"`
+	ContinueOnError bool   `long:"continue-on-error" description:"If proceeding protocols error, do not run following protocols (default: true)"`
+}
+
+// Validates the options sent to MultipleConfig, and parses the configFile
+func (x *MultipleConfig) Validate(args []string) error {
```

Comment:
The rational would be that someone could specify `./zgrab2 --config-file="mult.ini" http --port=8080`, which would make no sense. Either do a multiple scan, or do a single http. 

I think the naming here is probably the main issue, maybe instead of MultipleConfig it should be MultipleModules?